src/WalletNavigator.tsx

```diff
@@ -49,6 +53,16 @@ const WalletTabNavigator = () => {
           }}
         />
 
+        <Tab.Screen
+          name="nfts"
+          component={NftsNavigator}
+          options={{
+            tabBarIcon: ({focused}) => <Icon.Image size={28} color={focused ? '#17d1aa' : '#A7AFC0'} />,
+            tabBarLabel: strings.nftsTabBarLabel(nfts?.length ?? 0),
```

Comment:
is it on Figma?

Reply:
No, it was here before, and I just made it to compile. I haven't added any of the UI tweaks yet